conf/locale/ru/LC_MESSAGES/django.po

```diff
@@ -15798,11 +15798,18 @@ msgid ""
 "engaging, high-quality {platform_name} courses. Note that you will not be "
 "able to log back into your account until you have activated it."
 msgstr ""
+"Привет!"
+"Используйте нашу ссылку, чтобы активировать свою учетную запись и получить доступ к курсам MAMA."
```

Comment:
Is "MAMA" supposed to be hardcoded? I see that the english version uses {platform_name} instead. Shouldn't the russian version also use some variable?

Reply:
Fixed